Update tests result in UnicodeDecodeError after restart

Running the background update test on Windows 10 for Firefox 65.0b* to 66.0b3 is failing with a UnicodeDecodeError. I wasn't seeing this before 66.0b3 was pushed live so I wonder if this is a regression.

Here is the full output:
```
(iris-R3GGJVZj) C:\Users\iris\iris>iris -t background_update -f 65.0b10 -o
2019-01-28 14:10:37 [INFO] [mozdownload.scraper] Retrieving list of candidate builds from https://archive.mozilla.org/pub/firefox/candidates/65.0b10-candidates/
2019-01-28 14:10:38 [INFO] [mozdownload.scraper] Found 1 build: build1
2019-01-28 14:10:38 [INFO] [mozdownload.scraper] Selected build: build1
2019-01-28 14:10:38 [INFO] [mozdownload.scraper] Downloading from: https://archive.mozilla.org/pub/firefox/candidates/65.0b10-candidates/build1/win64/en-US/Firefox%20Setup%2065.0b10.exe
2019-01-28 14:10:38 [INFO] [mozdownload.scraper] Saving as: C:\Users\iris\.iris\cache\firefox-65.0b10-build1.en-US.win64.exe
100% |##########################################################################################################################################################################################################| Time:  0:00:04   9.6 MiB/s
2019-01-28 14:10:48 [INFO] [mozdownload.scraper] Retrieving list of candidate builds from https://archive.mozilla.org/pub/firefox/candidates/66.0b3-candidates/
2019-01-28 14:10:48 [INFO] [mozdownload.scraper] Found 1 build: build1
2019-01-28 14:10:48 [INFO] [mozdownload.scraper] Selected build: build1
2019-01-28 14:10:48 [INFO] [iris.firefox.firefox_app] Latest available version for beta channel is: 66.0b3
2019-01-28 14:10:48 [INFO] [iris.test_runner] Running tests
2019-01-28 14:10:48 [INFO] [iris.test_runner]
------------------------------------------------------------------------------------------------------------------------
2019-01-28 14:10:48 [INFO] [iris.test_runner] Executing: 1 - [background_update]: This is a test for Firefox background update.
2019-01-28 14:10:53 [SUCCESS] [iris.test_case] >>> PASSED <<< Step 1: Firefox version is correct (65.0b10)
2019-01-28 14:10:53 [INFO] [iris.test_case] Current version: 65.0b10, updating to version: 66.0b3.
2019-01-28 14:11:49 [ERROR] [iris.test_case] >>> ERROR <<< Error encountered in test

----------------------------------------------------------------------------------------------------------
» Traceback (most recent call last):                                                                     «
»   File "c:\users\iris\iris\iris\test_runner.py", line 69, in run                                       «
»     current.run()                                                                                      «
»   File "c:\users\iris\iris\iris\tests\update\background_update.py", line 67, in run                    «
»     'Firefox successfully updated from %s to %s.' % (current_version, update_step))                    «
»   File "c:\users\iris\iris\iris\asserts.py", line 49, in assert_contains                               «
»     Result('FAILED', message, actual, expected, print_error(format_stack(traceback.extract_stack())))) «
»   File "c:\users\iris\iris\iris\test_case.py", line 74, in add_results                                 «
»     '\n' + result.error if result.error else ''))                                                      «
» UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 109: ordinal not in range(128)    «
----------------------------------------------------------------------------------------------------------

2019-01-28 14:11:50 [INFO] [iris.api.helpers.results] [background_update] - >>> ERROR <<< (Finished in 60.87 second(s))

2019-01-28 14:11:50 [INFO] [iris.api.helpers.results]
------------------------------------------------------------------------------------------------------------------------
Platform: win, Firefox Version: 65.0, Firefox Build: 20190110221328
Passed: 0, Failed: 0, Skipped: 0 (Blocked: 0), Errors: 1 -- Total: 1                          Total time: 62.6 second(s)

The following tests did not pass:
background_update


------------------------------------------------------------------------------------------------------------------------
```

--- a/iris/api/helpers/results.py
+++ b/iris/api/helpers/results.py
@@ -68,10 +68,10 @@ def print_error(error):
     for line in lines:
         if len(line) > max_len:
             max_len = len(line)
-    result = result + '-' * (max_len + 4) + '\n'
+    result = result + '-' * (max_len + 6) + '\n'
     for line in lines:
         result = result + '>> ' + line + ' ' * (max_len - len(line)) + ' <<' + '\n'
-    result = result + '-' * (max_len + 4) + '\n'
+    result = result + '-' * (max_len + 6) + '\n'
     return result
 
 

--- a/iris/api/helpers/results.py
+++ b/iris/api/helpers/results.py
@@ -70,7 +70,7 @@ def print_error(error):
             max_len = len(line)
     result = result + '-' * (max_len + 4) + '\n'
     for line in lines:
-        result = result + '» ' + line + ' ' * (max_len - len(line)) + ' «' + '\n'
+        result = result + '>> ' + line + ' ' * (max_len - len(line)) + ' <<' + '\n'
     result = result + '-' * (max_len + 4) + '\n'
     return result
 

